developer-cli/Installation/Configuration.cs

```diff
@@ -10,6 +10,9 @@ public static class Configuration
 {
     public static readonly bool IsWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
     public static readonly bool IsMacOs = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
+    public static readonly bool IsLinux = RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
+    public static readonly string? WSLDistroName = Environment.GetEnvironmentVariable("WSL_DISTRO_NAME");
+    public static readonly bool IsWSL = IsLinux && !string.IsNullOrEmpty(WSLDistroName);
```

Comment:
`IsWSL` should be `IsWsl`, which is a naming convention in .NET. E.g. `SQL` is `Sql`.